components/ProfilePicChanger.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable prettier/prettier */
```

Comment:
This file shouldn't be put in `components` directory. Instead, create a directory named `screens/ProfileScreen` and put the current `ProfileScreen.js` file and all of other components which are used only by the screen in the directory, as it's a good chance that this `ProfilePicChanger` component won't be used anywhere else.

You may have a look at HomeScreen directory for reference of folder structure